atomdb/species.py

```diff
@@ -38,6 +38,11 @@
 from numbers import Integral
 
 elements_hdf5_file = files("atomdb.data").joinpath("elements_data.h5")
+datasets_hdf5_file = files("atomdb.data").joinpath("datasets_data.h5")
```

Comment:
The file `datasets_data.h5` should be placed under `atomdb/datasets`  intead of `atomdb/data`

As a note for the final stage of atomdb refactor, when setting the path where the dataset HDF5 is going to be stored, we should support custom paths; for example if the user wants to compile the dataset outside the atomdb package.